napari/components/viewer_model.py

```diff
@@ -467,7 +460,7 @@ def add_volume(
         opacity=1,
         blending='translucent',
         visible=True,
-        spacing=None,
+        scale=None,
```

Comment:
I'm not sure we want to rename this...?

Reply:
the reason i renamed is that `scale` is a property of all our layers - i'll probably move the keyword arg to the base layer (along with exposing `translate`), in prepartation for merging the volume layer with the image layer. at that point i felt like it was confusing to have `spacing` for non-volumetric stuff, whereas `scale` was clear regardless